### PR TITLE
Code submission tests

### PR DESCRIPTION
**Pull request by eledavi**
Corrected variable IMPORT_PATTERN so line 86 of CodeSubmissionObjectFactory.js evaluates to true and lines 87-93 are run

### CODE REVIEW

File: client/domain/CodeSubmissionObjectFactory.js [hunk 1]
```diff
@@ -164,8 +164,7 @@ tie.factory('CodeSubmissionObjectFactory', [
       var insertPos = 0;
       for (var i = 0; i < this._preprocessedCodeLines.length; i++) {
         if (this.IMPORT_PATTERN.test(this._preprocessedCodeLines[i])) {
-          var importLine = this._preprocessedCodeLines[i].slice(
-              this.TAB_LENGTH);
+          var importLine = this._preprocessedCodeLines[i].slice(0);
```
Comment (seanlip) [inline client/domain/CodeSubmissionObjectFactory.js]: Are you sure this change is correct? Did you test manually?

Comment (eledavi) [inline client/domain/CodeSubmissionObjectFactory.js]: I retested it and corrected it. New commit should be good.

File: client/domain/CodeSubmissionObjectFactory.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ tie.factory('CodeSubmissionObjectFactory', [
      * @type {RegExp}
      */
     CodeSubmission.prototype.IMPORT_PATTERN = new RegExp(
-      '^\\ {4}import\\ \\w+$');
+      '^\\ {0}import\\ \\w+$');
```
Comment (theresa-lee) [inline client/domain/CodeSubmissionObjectFactory.js]: Per our conversation, {0} is a no op.

**Comment by eledavi** [comment]
I corrected the regex variable. @theresa-lee

**Comment by theresa-lee** [comment]
Turns out that part of the code doesn't actually need to be tested anymore because that function is not necessary (being deleted in #358).

**Pull request closed without merging**